src/components/Editor/Editor.css

```diff
@@ -51,6 +51,23 @@ html[dir="rtl"] .editor-mount {
   direction: ltr;
 }
 
+:not(.empty-line):not(.new-breakpoint) > .CodeMirror-gutter-wrapper:hover {
+  width: 60px !important;
```

Comment:
I'm slightly concerned with all of the `!important` declarations -- can we adjust our selector specificity so we don't need them?

Reply:
this might be hard to fix.. but yes, we should try

Reply:
i see. makes sense